### DISCUSSION TITLE
Which Genie Plan?

### DISCUSSION THREAD

**Discussion by kpcenti** [Slyyxp/rsack#58] [Q&A] [answered]
Genie seems to separate their subscriptions with Mobile, PC, or Mobile&PC unlimited streaming, Would the mobile only one be sufficient to use this tool?

**Comment by Slyyxp** [comment, marked as answer]
This is the plan I use personally: https://product.kt.com/wDic/productDetail.do?ItemCode=1282

Otherwise you're probably looking for either `초고음질 음악감상` or `리얼지니팩 멤버십`
https://www.genie.co.kr/buy/geniePack
Both of those plans are tagged with `5G고음질전용관` which I do believe is their indicator for FLAC support.

The KT exclusive plans are higher priced which could indicate that only they support 24bit. `초고음질 음악감상` may only be 16bit, I have absolutely no idea.

If all you are interested in is MP3 then I suggest just grabbing one of the cheaper unlimited streaming plans.

Sorry that I could not point you to one specific plan definitively, for those not with KT `초고음질 음악감상` is probably your best bet.

**Reply by kpcenti** [reply]
I was mostly interested in FLAC / Lossless. I'm unfortunately not a Korean resident, so I'll have to get it through some other means. I'll have to look in to which 30 day plan has FLAC

**Reply by Slyyxp** [reply]
I see, unfortunately I can't help further with this as I'm unsure how you would go about purchasing a FLAC subscription outside of Korea.

There's the chance you can utilize the mobile app for this if you already have login credentials but I cannot say for certain and do not know which plans are available through these means either.

Sorry I can't be of further help, good luck.
